app/models/game.rb

```diff
@@ -1,7 +1,17 @@
 class Game < ActiveRecord::Base
+  attr_accessor :creator_plays_as_black # for checkbox access
   has_many :pieces, dependent: :destroy
-  has_many :users
-  validates :game_name, :presence => { :message => "Game name is required!" }
+  belongs_to :white_user, class_name: 'User', foreign_key: :white_user_id
+  belongs_to :black_user, class_name: 'User', foreign_key: :black_user_id
+  validates :game_name, presence: { :message => "Game name is required!" }
+  validates :white_user_id,
+            presence: true, unless: :black_user_id?, on: :create
+  validates :black_user_id,
+            presence: true, unless: :white_user_id?, on: :create
+  validate :user_id_exists, on: :create
+  validate :white_user_id_exists, if: :white_user_id?, on: :update
+  validate :black_user_id_exists, if: :black_user_id?, on: :update
```

Comment:
these three validation methods check to see that the users are valid (exist) since currently there aren't foreign keys